docs/admin/troubleshooting.md

```diff
@@ -326,112 +326,407 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 ## Common Errors and Workarounds
 
 ### Use dmg command without daos_admin privilege
+```
+# Error message or timeout after dmg system query
+$ dmg system query
+ERROR: dmg: Unable to load Certificate Data: could not load cert: stat /etc/daos/certs/admin.crt: no such file or directory
+
+# Workaround
+
+# 1. Make sure the admin-host /etc/daos/daos_control.yml is correctly configured.
+	# including:
+		# hostlist: <daos_server_lists>
+		# port: <port_num>
+		# transport\config:
+			# allow_insecure: <true/false>
+			# ca\cert: /etc/daos/certs/daosCA.crt
+			# cert: /etc/daos/certs/admin.crt
+			# key: /etc/daos/certs/admin.key
+
```

Comment:
the error states that the admin.crt file is not found. so the fix should be to ensure that this certificate file exists and is readable... checking the daos_control.yml is all well as a prereq step, but what's shown here is all commented out so certainly not a useful example... also some underscores got lost (e.g. ca\cert instead of ca\_cert). Tip: in markDown, if you wrap the yml into a code block with 3 backticks, you will not need to escape the underscores...

Reply:
I just changed the formatting only and not the content to aligned with the other file format suggested by Johann.
Better to view this https://github.com/daos-stack/daos/pull/8085/commits/bba574816f0da6beb431ba3743c81247fdc6061c patch as some how combined view on "File changed" is not showing the all the document difference. So it's not new addition in line but just the line format editing.
But I will change this to ca_cert